File: 2/Python/Hardware.py
from Product import Product

class Hardware(Product):
	# Konstruktor
	def __init__(self):
		self.__brand = ""
		self.__model = ""

	# Set & Get

	# Brand
	def setBrand(self, brand):
		self.__brand = brand

	def getBrand(self):
		return self.__brand

	# Model
	def setModel(self, model):
		self.__model = model

	def getModel(self):
		return self.__model

File: 1/Hardware.java
public class Hardware extends Product{
	// Atribut
	private String brand;	// Brand
	private String model;	// Model

	// Konstruktor
	public Hardware() {
		brand = "";
		model = "";
	}

	/* Set & Get */

	// Brand
	public void setBrand(String brand) {
		this.brand = brand;
	}
	public String getBrand() {
		return this.brand;
	}

	// Model
	public void setModel(String model) {
		this.model = model;
	}
	public String getModel() {
		return this.model;
	}
}